Include/internal/pycore_code.h

```diff
@@ -64,7 +64,7 @@ typedef struct {
     uint16_t counter;
     uint16_t type_version[2];
     uint16_t keys_version[2];
```

Comment:
We can shrink this even further by reducing type_version and keys_version in the future.